src/sedcli_main.c

```diff
@@ -404,6 +424,20 @@ int write_mbr_handle_opts(char *opt, char **arg)
 	return 0;
 }
 
+int blocksid_handle_opts(char *opt, char **arg)
+{
+	int hwreset_flag = 0; /* No reset BlockSID upon power events */
+
+	if (!strcmp(opt, "device")) {
+		strncpy(opts->dev_path, arg[0], PATH_MAX - 1);
+	} else if (!strcmp(opt, "hwreset")) {
+		hwreset_flag = atoi(arg[0]);
+		opts->hardware_reset = (hwreset_flag == 1) ? true : false;
```

Comment:
This can be changed into 1 line after changing the hardware reset option to flag.
opts->hardware_reset = true;

Reply:
See my previous comment, not intended to use this as a flag to indicate setting hardware reset or not.